src/query.h

```diff
@@ -15,6 +15,6 @@
  * tuples, either until #SQLITE_DONE is returned or a full page of the given
  * buffer is filled.
  */
-int query__batch(sqlite3_stmt *stmt, struct buffer *buffer);
+int query_batch(sqlite3_stmt *stmt, struct buffer *buffer);
```

Comment:
Since this is not a public symbol (exported in `dqlite.h`), but rather an internal one it should be `queryBatch`.

Reply:
Fixed, there was a namespace clash here I had to fix first with s/queryBatch/gatewayQueryBatch/g